zerver/openapi/zulip.yaml

```diff
@@ -2832,18 +2822,19 @@ paths:
                                   items:
                                     type: integer
                                 all:
+                                  deprecated: true
                                   type: boolean
                                   description: |
-                                    Whether the flag was removed from all messages.
-                                    If this is true then the `messages` array will be empty.
+                                    Will be `false` for all specified flags.
+
+                                    **Deprecated** and will be removed in a future release.
                                 message_details:
                                   description: |
-                                    Present if `message` and `update_message_flags` are both present in
-                                    `event_types` and the `flag` is `read` and the `op` is `remove`.
+                                    Only present if the specified `flag` is `"read"`.
```

Comment:
This "Present if X are present in `event_types` is what we use in the register response documentation and not the events documentation.